deploy

```diff
@@ -4,7 +4,14 @@
 # has been bumped, otherwise you will need to manually restart all
 # the pods with kubectl.
 #
-# NOTE: must be run from the compenent's root directory.
+# NOTE: must be run from the component's root directory.
```

Comment:
Or.... rather than muddying the waters with a bunch of platform specific hacks and control flow, scould we not just change the expression to be POSIX compliant so it just works regardless of platform? e.g.

```bash
echo 'https://github.com/unikorn-cloud/ui' | sed --posix -e 's,https://,,'
```

Just drop the `--posix` and it should just work on crapOS

Reply:
Yeah, fair point

Worth noting that that would remove the conditional regex of http|https. Are you happy to remove the http check since it's unlikely to ever be an http endpoint? It was my first thought to be honest as it's very, very unlikely it'll ever need to replace http, it'd be an eaiser fix to do that ^

Reply:
I'm shocked you of all people think http is still a thing 😄

Reply:
I know but the original sed accounted for it :-D